src/commons/locations.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
There's specific comments below, but in general for JSONs that are made as "configs" or single sources of truth.

LHS: Key is supposed to be the user friendly proxy we use as devs to make our code look concise.
RHS: Value is supposed to be the actual text (in whatever form, ie snake or camel case or number or hashcode even) which we use to transfer data between services in a consistent way.
Both should probably not have spaces in order to be generally applied.

If you want to have spaces as a "pretty form" feel free to do something like 
````
{
'LA' : {
    canonical : 'Los_Angeles',
    pretty : 'Los Angeles'
}
}
````
So LA is useful and short variable for developers, canonical is the true name we pass around, and pretty is used for display. 
If you wonder if it's redundant to put something like `Seattle : Seattle` , that's sadly a limitation of JSON in my mind. But It helps us create psudo enums bc if every developer references this locations.json, they will only have a discrete set of locations to choose from which can be updated.

Hope that makes sense :)

Reply:
Ok just to be clear. `Seattle :  Seattle ` is still the format we're going to go forth with?